rotkehlchen/externalapis/etherscan.py

```diff
@@ -545,7 +535,7 @@ def get_token_transaction_hashes(
                 except DeserializationError as e:
                     log.error(
                         f"Failed to read transaction timestamp {entry['hash']} from {chain_id} "
-                        f'etherscan for {account} in the range {from_ts} to {to_ts}. {e!s}',
+                        f'etherscan for {account} in the range {from_block} to {to_block}. {e!s}',
```

Comment:
Isn't this goign to make logs reading harder? Are we logging somewhere above in the call the from/to ts?